src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -24,6 +24,7 @@ type Props<T: string, U: {type: T}> = {|
     onChange: (value: Array<U>) => void,
     onSettingsClick?: (index: number) => void,
     onSortEnd?: (oldIndex: number, newIndex: number) => void,
+    onTriggerMessage?: (message: Message) => void,
```

Comment:
I don't think `trigger` is the correct term for this use case, but I also can't find another one that fits better ...

Reply:
I even didnt wanted to use "on" here but our eslint rule doesnt allow it. Alternative to trigger would be onDispatchMessage. What do you think?

Reply:
i think would go with `onShowSnackbar` or `onDisplaySnackbar`

Reply:
I would go with `onDisplaySnackbar` then @luca-rath what do you think?